Model/src/main/java/org/gusdb/wdk/model/report/config/AnswerDetails.java

```diff
@@ -29,6 +29,7 @@ public enum AttributeFormat {
   private List<SortDirectionSpec<AttributeField>> _sorting = new ArrayList<>();
   private ContentDisposition _contentDisposition = ContentDisposition.INLINE;
   private AttributeFormat _attributeFormat = AttributeFormat.DISPLAY;
+  private boolean _isBufferEntireResponse = false; // default to standard streaming model
```

Comment:
add a comment indicating why we have this flag, and what the approach we take is.

Reply:
I think the existing docs in AnswerDetailsReporter are enough.  As as developer, if I wondered about this flag, my first question would be "where it it used?" which would lead directly to where there is ample documentation.